src/styles/partials/_theme.scss

```diff
@@ -0,0 +1,169 @@
+body{
+	color: $primary-color;
+	margin: 0;
+	padding: 0;
+	width: 100vw;
+	min-height: 100vh;
+	background: #fff;
+	overflow-x: hidden;
+}
+
+a {
+	color: $link-color;
+}
+
+a:hover {
+	color: $link-hover-color;
+	text-decoration: underline;
+	cursor: pointer;
+}
+
+hr {
+	margin-top: 20px;
+	margin-bottom: 20px;
+	border: 0;
+	border-top: 1px solid $line-color;
+}
+
+
+
+/****HOME PAGE SEARCH BOX ELEMENTS****/
+
+
+.input-group {
+	display: table;
+}
+
+.input-group-btn {
+	display: table-cell;
+	width: 1%;
+}
+
+.btn {
+	border: 1px solid transparent;
+	display: inline-block;
+	padding: 6px 12px;
+	vertical-align: middle;
+	border-radius: 4px;
+	border-top-left-radius: 0px;
+	border-bottom-left-radius: 0px;
+}
+
+/****NAVBAR ELEMENTS****/
```

Comment:
Well I guess then move navbar to its own Module. like Footer (`<app-footer>`). With its own component (`<app-navbar>`) which will contain its own styles.